src/content/packages-and-plugins/swift-package-manager/for-app-developers.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Swift Package Manager for app developers
+description: How to use Swift Package Manager for native iOS or macOS dependencies
+---
+
+:::warning
+Flutter is migrating to [Swift Package Manager][] to manage iOS and macOS native
+dependencies.
+This is an experimental feature that might change in the future.
+It is currently only available on the [`main` channel][].
+Flutter will continue to support CocoaPods until further notice.
+:::
+
+Flutter's Swift Package Manager integration has several benefits:
+
+1. **Access to the Swift package ecosystem**.
+   Flutter plugins can use the growing ecosystem of [Swift packages][]! 
+2. **Simplifies Flutter installation**.
+   Swift Package Manager is bundled with Xcode.
+   In the future, you won’t need to install Ruby and CocoaPods to target iOS or
+   macOS.
```

Comment:
**issue:** 
- Use parallel structure in your list sentences.
- Remove the exclamation point per [GSG](https://developers.google.com/style/periods#exclamation-points).
- Keep the documentation timeless.
- Use present tense.

```suggestion
1. **Provides access to the Swift package ecosystem**.
   Flutter plugins can use the growing ecosystem of [Swift packages][]! 
1. **Simplifies Flutter installation**.
   Xcode includes Swift Package Manager.
   Targeting iOS or macOS doesn't require Ruby and CocoaPods.
```

Reply:
This sounds good but I tweaked the last sentence:

```diff
- Targeting iOS or macOS doesn't require Ruby and CocoaPods.
+ You don't need to install Ruby and CocoaPods if your project uses
+ Swift Package Manager.
```

Let me know what you think!